Directory.Build.targets

```diff
@@ -19,19 +19,12 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
-    <!--
-      Reset PackageOutputPath property from arcade as we need to use ConfigurationGroup instead of Configuration which arcade uses.
-      See issue https://github.com/dotnet/corefx/issues/33275
-    -->
-    <PackageOutputPath>$(ArtifactsDir)packages/$(ConfigurationGroup)/</PackageOutputPath>
-    <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)symbols/</SymbolPackageOutputPath>
+    <ArtifactsPackagesDir>$([MSBuild]::NormalizePath('$(ArtifactsDir)', 'packages/', '$(ConfigurationGroup)'))</ArtifactsPackagesDir>
+    <ArtifactsShippingPackagesDir>$([MSBuild]::NormalizePath('$(ArtifactsPackagesDir)', 'Shipping/'))</ArtifactsShippingPackagesDir>
```

Comment:
Will change to `NormalizeDirectory` once my demo build is done to not mess up the commit sha's as it is needed for publishing.

Reply:
please also remove the slash in the L22 in 'packages/'.

Reply:
and I suppose you will change L24 as well.

Reply:
Correct.